### DISCUSSION TITLE
problem on lxc mqtt

### DISCUSSION THREAD

**Discussion by titof2375** [tteck/Proxmox#726] [Help Needed] [answered]
Hello I wanted to install the lxc mqtt which is installed without problem, on the other hand for this command that I made in the lxc:
sudo mosquitto_passwd -c /etc/mosquitto/passwd <usr>, I got this error
![mqtt](https://user-images.githubusercontent.com/31250165/201516247-fa70c7e4-a1c5-4a67-8d51-c42182928a98.PNG)

**Comment by tteck** [comment, marked as answer]
`sudo mosquitto_passwd -c /etc/mosquitto/passwd titof` omit the "< >"

**Comment by titof2375** [comment]
thank you
